core/src/main/java/de/danoeh/antennapod/core/cast/CastConsumerImpl.java

```diff
@@ -0,0 +1,10 @@
+package de.danoeh.antennapod.core.cast;
+
+import com.google.android.libraries.cast.companionlibrary.cast.callbacks.VideoCastConsumerImpl;
+
+public class CastConsumerImpl extends VideoCastConsumerImpl implements CastConsumer {
```

Comment:
Usually better names for classes like this (since there could be other implementations) is 'DefaultInterfaceName'.

So this would be 'DefaultCastConsumer'.